### PR TITLE
[FIX] l10n_fi: set the right tags for the finnish accounts

### PR DESCRIPTION
**Pull request by mafo-odoo**
Reproduction :
Install "Finnish Localization", go on chart of accounts and add tag_ids on the tree view.

Current behavior before PR:
There are some mismatches between the tags and the accounts

Desired behavior after PR is merged:
Tags are modified/created/deleted to have no mismatch and to correspond to Finnish accounting

opw-2702912

### CODE REVIEW

**Comment by mafo-odoo** [comment]
@jco-odoo

**Comment by jco-odoo** [comment]
@vin-odoo Could you have a look with the changes to account codes you did in later versions to see how it compares?

**Comment by vin-odoo** [comment]
@jco-odoo @mafo-odoo I think we have two issues here.
The tags that could be wrong (fixed here, I don't know much about them). It won't have an impact on the reports (if the code is up to date) as we don't use them anymore to calculate it.

But there is (at least) another issue the client reported. Line B) II has the wrong formula (my bad)

Formula is as such:

```xml
<field name="formulas">FIBS_ASS_BII_1 + FIBS_ASS_BII_2 + FIBS_ASS_BIII + FIBS_ASS_BIV</field>
```
as you can see, the formula is using the B III and B IV lines to calculate B II. This is definitely wrong.

**Comment by mafo-odoo** [comment]
@vin-odoo indeed there is this other bug but it was not in the synthesis of Mélanie, do you think the modifications for this bug should be added on this pr? or it should be subject of a new PR and/or task?

**Comment by vin-odoo** [comment]
@mafo-odoo Add it here yes, to me it is related to the same ticket and it makes things easier to have it in one place.
Eventually, you could make two separate commits for the issues if you think it's better.

**Comment by mafo-odoo** [comment]
@vin-odoo the report you mentioned is in enterprise/l10n_fi_reports so we are gonna need a new PR anyway ^^

**Comment by mafo-odoo** [comment]
@jco-odoo @vin-odoo is this PR ok for you now?

**Comment by mafo-odoo** [comment]
@jco-odoo @vin-odoo the PR on enterprised has been merged, could you add an r+

**Comment by jco-odoo** [comment]
@mafo-odoo Why do you add Cash and Bank accounts?  These should be added automatically based on prefix normally.

**Comment by mafo-odoo** [comment]
@jco-odoo to add a tag "Tase: Rahat ja pankkisaamiset" to them

**Comment by mafo-odoo** [comment]
@jco-odoo

**Comment by mafo-odoo** [comment]
@jco-odoo @vin-odoo

File: addons/l10n_fi/data/account.account.template.csv [hunk 1]
```diff
@@ -229,3 +229,7 @@
 "account_9970","9970","Laskennalliset verot","account.data_account_type_expenses","l10n_fi.fi_chart_template","False","l10n_fi.account_tag_taxes_for_the_period"
 "account_9980","9980","Muut välittömät verot","account.data_account_type_expenses","l10n_fi.fi_chart_template","False","l10n_fi.account_tag_taxes_for_the_period"
 "account_9990","9990","Vähemmistöosuudet","account.data_account_type_expenses","l10n_fi.fi_chart_template","False",
+"account_19101","19101","Cash","account.data_account_type_liquidity","l10n_fi.fi_chart_template","False","l10n_fi.account_tag_liquidity"
```
Comment (jco-odoo) [inline addons/l10n_fi/data/account.account.template.csv:232]: I really don't think we need this.

Comment (jco-odoo) [reply addons/l10n_fi/data/account.account.template.csv]: No, we need at least one receivable/payable and a receivable for the PoS

**Comment by jco-odoo** [comment]
@vin-odoo Could you still check as well?

File: addons/l10n_fi/data/account.account.template.csv (hunk 1 above)
Comment (vin-odoo) [reply addons/l10n_fi/data/account.account.template.csv]: Outstanding receipts and payments accounts should be automatically created by the bank/cash journals when they are created at the installation of the chart template.

**Comment by mafo-odoo** [comment]
@vin-odoo up

**Comment by vin-odoo** [comment]
@mafo-odoo replied here https://github.com/odoo/odoo/pull/88198#discussion_r869873098

**Comment by mafo-odoo** [comment]
@jco-odoo @vin-odoo

**Comment by jco-odoo** [comment]
@mafo-odoo A bit weird as the tags are not used anymore in the reports, but good to fix anyways.

**Pull request closed without merging**